<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HeaderBarSearch</class>
 <widget class="QWidget" name="HeaderBarSearch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>24</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>22</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>24</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">HeaderBarSearch:hover{
	border:green 1px solid;
}
HeaderBarSearch:focus{
	border:red 1px solid;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLineEdit" name="searchTextEdit">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>22</height>
        </size>
       </property>
       <property name="toolTip">
        <string>请输入查询内容</string>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgb(255, 255, 255)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="returnButton">
       <property name="minimumSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
       <property name="toolTip">
        <string>返回</string>
       </property>
       <property name="statusTip">
        <string>返回</string>
       </property>
       <property name="whatsThis">
        <string>返回</string>
       </property>
       <property name="styleSheet">
        <string notr="true">background:rgba(0, 0,0,0);
border:none;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/DictSideBar/res/img/return.png</normalon>
        </iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="allTextSearchButton">
       <property name="minimumSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
       <property name="toolTip">
        <string>全文检索</string>
       </property>
       <property name="statusTip">
        <string>全文检索</string>
       </property>
       <property name="whatsThis">
        <string>全文检索</string>
       </property>
       <property name="styleSheet">
        <string notr="true">background:rgba(0, 0,0,0);
border:none;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/res/img/allTextSearch_off.png</normaloff>
         <normalon>:/DictSideBar/res/img/item/allTextSearch_off.png</normalon>:/res/img/allTextSearch_off.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="showParagraph">
       <property name="minimumSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
       <property name="toolTip">
        <string>显示词条内容</string>
       </property>
       <property name="statusTip">
        <string>显示词条内容</string>
       </property>
       <property name="whatsThis">
        <string>显示词条内容</string>
       </property>
       <property name="styleSheet">
        <string notr="true">background:rgba(0, 0,0,0);
border:none;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/DictSideBar/res/img/item/showParagraph_off.png</normalon>
        </iconset>
       </property>
       <property name="autoRepeatDelay">
        <number>200</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
